File: Course_01/C++ Language/EstruturaCondicional/tempo_de_jogo.cpp
#include <iostream>

using namespace std;

int main()
{
    int horaInicial, horaFinal, duracao;

    cout << "Hora inicial: ";
    cin >> horaInicial;
    cout << "Hora final: ";
    cin >> horaFinal;

    if (horaInicial == horaFinal) {
        duracao = 24;
    }
    else if (horaInicial >= 12) {
        duracao = 24 - horaInicial + horaFinal;
    }
    else {
        duracao = horaFinal - horaInicial;
    }

    cout << "O JOGO DUROU " << duracao << " HORA(S)" << endl;
    return 0;
}

File: Course_01/Python/EstruturaCondicional/tempo_de_jogo.py
horaInicial = int(input("Hora inicial: "))
horaFinal = int(input("Hora final: "))

if horaInicial == horaFinal:
    print("O jogo durou 24 Hora(s)")
elif horaInicial > horaFinal:
    print(f"O jogo durou {(24 - horaInicial) + horaFinal} Hora(s)")
elif horaInicial < horaFinal:
    print(f"O jogo durou {horaFinal - horaInicial} Hora(s)")